letsdrill.rb

```diff
@@ -1,24 +1,34 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  letter_grade = ""
+  if integer >= 90
+    letter_grade = "A"
+  elsif integer >= 80
+    letter_grade = "B"
+  elsif integer >= 70
+    letter_grade = "C"
+  elsif integer >= 60
+    letter_grade = "D"
+  else
+    letter_grade = "F"
```

Comment:
The whole `if`/`elsif`/`else` structure evaluates to the value of whichever branch it takes. So, you could move the local variable assignment out of each condition like so:

```ruby
letter_grade = if integer >= 90
                 "A"
               elsif ...
```